scootapi/server/runJob.go

```diff
@@ -0,0 +1,93 @@
+package server
```

Comment:
done, and i will do another code review to make the codebase consistent with this naming convention, since its mostly me who messed it up :)